script/Ciel/Day 7/07_01_CIEL07_2_1.txt

```diff
@@ -1,30 +1,33 @@
 [sha:f708d3b834a071bf47d8f1e2a2ae669637b21a29]{
 -- Page 10008, Offset 15206.
 -- やはりシエル先輩の茶道室に行こう。
-I should go to the tea ceremony room where Ciel is after all.
+I think heading to the tea ceremony room to see Ciel is the right call after all.
 }
 [sha:6224ee4d43eebe10dcd27b02933cbef7f2a71594]{
 -- Page 10008, Offset 15207.
 -- ノエル先生はいまいち緊張感がないというか、話をはぐらかすところがある。
-Noel is kind of too relaxed――maybe prone to talking a bit evasively.
+Noel-sensei is a little too relaxed, and she seems to have a habit of dodging questions.
 }
 [sha:f58218b3cc13c11b1ca4a4386eeefd104dce921a]{
 -- Page 10008, Offset 15208.
 -- なぜ彼女たちが街に残っているのか―――
-Why do the two of them still remain in the city?
+Why are the two of them still here in this city?
 }
 [sha:7f0a992ec5e22708f761a92ce2f01d075a4711a8]{
 -- Page 10008, Offset 15209.
 -- それをはっきりさせない事には安心して家に帰れない。
-I can't return home with peace of mind if I don't get a definitive answer to that question.
+// Inverted for flow
+Unless I get a definitive answer to that question,
+ it's going to bug me all the way home.
```

Comment:
hm, not sure I like this change because it reads with a bit of a different, almost lighter nuance. 
New structure is nice so maybe like "I can't go home satisfied" instead. Or revert that bit to
> Unless I get a definitive answer to that question, I can't go home with peace of mind.